matrix/all-in-one.yaml

```diff
@@ -0,0 +1,575 @@
+    type: int
+    category: instrumentation
+    toDoComment: int?
+  - name: OTEL_DOTNET_AUTO_GRAPHQL_SET_DOCUMENT
+    description: Whether the GraphQL instrumentation can pass raw queries as a `graphql.document` attribute. As queries might contain sensitive information, the default value is `false`.
+    default: 'false'
+    type: boolean
+    category: instrumentation
+  - name: OTEL_DOTNET_AUTO_TRACES_ADDITIONAL_LEGACY_SOURCES
+    description: Comma-separated list of additional legacy source names to be added to the tracer at the startup. Use it to capture `System.Diagnostics.Activity` objects created without using the `System.Diagnostics.ActivitySource` API.
+    default: ''
+    type: string
+    category: instrumentation
+  - name: OTEL_LOG_LEVEL
+    description: Sets the logging level for instrumentation log messages. Possible values are `none`, `error`, `warn`, `info`, and `debug`. The default value is `info`. Can't be set using the web.config or app.config files.
+    default: 'info'
+    type: enum
+    category: diagnostic logging
+    toDoComment: handle enum values/mark as string
+  - name: OTEL_DOTNET_AUTO_LOG_DIRECTORY
+    description: Directory of the .NET tracer logs. The default value is `/var/log/opentelemetry/dotnet` for Linux, and `%ProgramData%\OpenTelemetry .NET AutoInstrumentation\logs` for Windows. Can't be set using the web.config or app.config files.
+    default: 'see descruotuin'
+    type: string
+    category: diagnostic logging
+    toDoComment: defaultvalue
+  - name: OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED
+    description: Whether the traces console exporter is activated. The default value is `false`.
+    default: 'false'
+    type: boolean
+    category: diagnostic logging
+  - name: OTEL_DOTNET_AUTO_METRICS_CONSOLE_EXPORTER_ENABLED
+    description: Whether the metrics console exporter is activated. The default value is `false`.
+    default: 'false'
+    type: boolean
+    category: diagnostic logging
+  - name: OTEL_DOTNET_AUTO_LOGS_CONSOLE_EXPORTER_ENABLED
+    description: Whether the logs console exporter is activated. The default value is `false`.The default value is `false`.
+    default: 'false'
+    type: boolean
+    category: diagnostic logging
+  - name: OTEL_DOTNET_AUTO_LOGS_INCLUDE_FORMATTED_MESSAGE
+    description: Whether the log state have to be formatted. The default value is `false`.
+    default: 'false'
+    type: boolean
+    category: diagnostic logging
+  - name: MANUAL SETTINGS
+    description: How to handle https://docs.splunk.com/observability/en/gdi/get-data-in/application/otel-dotnet/configuration/advanced-dotnet-configuration.html#environment-variables-for-manual-installation?
+    default: 'N/A'
+    type: N/A
+    category: manual
+    toDoComment: See description
+instrumentations:
+  - keys:
+      - ASPNET
+    instrumentedLibraries:
+      - name: ASP.NET Framework (.NET Framework)
+        supportedVersions: See :ref:`dotnet-otel-versions`
```

Comment:
Can we use snake casing e.g. `supported_versions` like [OTel schema](https://github.com/open-telemetry/opentelemetry-specification/blob/main/schemas/1.21.0) and [OTel config](https://github.com/open-telemetry/opentelemetry-configuration/blob/main/examples/kitchen-sink.yaml)

Reply:
it should be common decision snake/camel case/dashes, I can change it to underscores. Based on what I have read, there is no common yaml standard.

Reply:
>  Based on what I have read, there is no common yaml standard.

This is true. Thus, I just propose to use the YAML conventions used in OTel 😉

Reply:
fccd6eaad5e9da405bb441e41fa7a5a6cd3dd8a0